codenav/codenav_run.py

```diff
@@ -232,16 +233,18 @@ def main():
             raise ValueError("No playground_dir provided")
 
         if args.code_dir is None:
-            path_to_module = os.path.abspath(
-                os.path.dirname(importlib.import_module(args.module).__file__)
-            )
-            args.code_dir = os.path.dirname(path_to_module)
-
-            force_subdir = os.path.basename(path_to_module)
+            # path_to_module = os.path.abspath(
+            #     os.path.dirname(importlib.import_module(args.module).__file__)
+            # )
+            # args.code_dir = os.path.dirname(path_to_module)
+            args.code_dir = os.path.dirname(args.module)
```

Comment:
Hmm, i think i misunderstood the purpose of `args.module`. 

I was interpreting it as absolute path to a code directory which is a module. For example if i cloned the target codebase like the codenav repo to `/home/tanmay/codenav` and wanted to index contents of a module within that target repo like `/home/tanmay/codenav/codenav` such that the es document file paths are `codenav.environments.code_env` (rather than `environments.code_env`) then i would use:
```
--module /home/tanmay/codenav/codenav
```

On the other hand if i wanted to index all files in  `/home/tanmay/codenav` (including things outside the codenav submodule like `scripts`) then I would use:
```
--code_dir /home/tanmay/codenav
```

 I guess that isn't how you are thinking of the `--module` flag?

Reply:
I'm thinking of the module flag in the sense that you're attempting to use a module that's currently available via
```
import module
```

That way you can just do
```
pip install transformers
```

and then `--module transformers` will just work. This is what was implemented before the change.

Reply:
fixed